scripts/fuzz.sh

```diff
@@ -85,10 +87,13 @@ main()
   local BUILD_ONLY=false
   local FUZZER_STDOUT=/dev/stdout # bashism
   local TEST_DURATION=3
+  local IPC=3
```

Comment:
once this lands it will possibly break oss-fuzz

Please send a PR right after to fix https://github.com/google/oss-fuzz/blob/master/projects/sound-open-firmware/build.sh

Reply:
Thanks, I was afraid of something like this :-(

It will be even worse than "break" because `-DCONFIG_IPC_MAJOR_3=y` will _silently_ take precedence over `CONFIG_IPC_MAJOR_4` inside `-DEXTRA_CONF_FILE=stub_build_all_ipc4.conf`.

So this PR here is breaking backwards compatibility... I considered leaving some "undefined" IPC version in place but that would have added significant complexity.

There is one simpler thing I can do: drop the IPC=3 default and make either -3 or -4 required. So this will still break backwards compatibility but NOT silently. What do you think?

Reply:
That is fine, we go without a build for a day is not an issue. Backwards compatibility is not as much as an issue since we are worried about fuzzing going forward primarily and reproducibility rather than bisecability.

Reply:
> That is fine, we go without a build for a day is not an issue. 

You will go with a build. But the IPC4 fuzzing will actually fuzz IPC3 silently.

The more I think about it, the less I like it.

Reply:
> So this PR here is breaking backwards compatibility... I considered leaving some "undefined" IPC version in place but that would have added significant complexity.

So it wasn't that complex after all.

Full backwards compatibility now, zero regression.